x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/hooks/use_create_analytics_form/reducer.ts

```diff
@@ -222,6 +262,35 @@ export const validateAdvancedEditor = (state: State): State => {
   return state;
 };
 
+/**
+ * Validates provided MML isn't lower than the estimated one.
+ */
+export function validateMinMML(estimatedMml: string) {
+  return (mml: string) => {
+    // @ts-ignore
+    const mmlInBytes = numeral(mml.toUpperCase()).value();
```

Comment:
As soon as I set the job type in the flyout, the page goes blank with the following errors in the console:

![image](https://user-images.githubusercontent.com/7405507/75774971-f785f000-5d48-11ea-8abd-9fef04d807ea.png)